### PR TITLE
Bug 1777258: Changed how backingstore is spelled through out the console

### PR DESCRIPTION
**Pull request by bipuladh**
- Changed how a backing store is spelled in the console.

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx [hunk 1]
```diff
@@ -181,14 +181,14 @@ const BackingStorePage: React.FC<BackingStorePageProps> = React.memo(
         <Form className="nb-bc-step-page-form">
           <Title headingLevel="h3" size="xl" className="nb-bc-step-page-form__title">
             <Flex breakpointMods={[{ modifier: 'justify-content-space-between' }] as any}>
-              <FlexItem>Tier 1 - backingStore ({state.tier1Policy})</FlexItem>
+              <FlexItem>Tier 1 - backing Store ({state.tier1Policy})</FlexItem>
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx]:
```suggestion
              <FlexItem>Tier 1 - Backing Store ({state.tier1Policy})</FlexItem>
```

File: frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx [hunk 2]
```diff
@@ -229,7 +229,7 @@ const BackingStorePage: React.FC<BackingStorePageProps> = React.memo(
         {showTier2Table && (
           <Form className="nb-bc-step-page-form">
             <Title headingLevel="h3" size="xl">
-              Tier 2 - backingStore ({state.tier2Policy}){' '}
+              Tier 2 - backing Store ({state.tier2Policy}){' '}
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx]:
```suggestion
              Tier 2 - Backing Store ({state.tier2Policy}){' '}
```

File: frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx (hunk 1 above)
Comment (shirimordechay) [reply frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx]: Yes, the labels should start with a capital letter, the same as other entities in the console (such as Storage Class).

**Pull request merged**